toolkit/scripts/repoquerywrapper.mk

```diff
@@ -55,4 +55,3 @@ $(STATUS_FLAGS_DIR)/repoquerywrapper.flag: $(go-repoquerywrapper) $(chroot_worke
 		$(if $(filter y,$(ENABLE_TRACE)),--enable-trace) \
 		--timestamp-file=$(TIMESTAMP_DIR)/repoquerywrapper.jsonl && \
 	scripts/update-target-if-output-changed.sh $@ $(QUERY_OUTPUT_FILE)-old $(QUERY_OUTPUT_FILE)
-
```

Comment:
nit: Restore deleted empty line.

Reply:
Ah, my editor considers multiple newlines at the end to be trailing whitespace. Will re-save without autoformat